templates/vertical-full-page-map/markup/mobilelisttoggles.hbs

```diff
@@ -2,11 +2,11 @@
   <button class="Answers-mobileToggle Answers-mobileToggle--results js-locator-listToggle">
     <span class="Answers-viewListText">
       {{> listIcon}}
-      List
+      {{ translate phrase="List" context="Toggle on locator to see results list view" }}
```

Comment:
I don't think the translators will know what a 'locator' is. Sometimes for context we simply give them a an example usage of the word. So what do you think of switching the context to 'A list of results'?